summary/summary.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+from lightning import Plugin, Millisatoshi
+import json
+import requests
```

Comment:
The dependency on `requests` should be in a `requirements.txt` file. Probably also should list `pylightning` as a dependency.

Reply:
I guessed >=0.0.6 for pylightning, but I guess that's OK?

requests>=2.0.0 seemed reasonable, too.